guide/how-it-works/lightning-services.md

```diff
@@ -0,0 +1,325 @@
+
+Backing up your lightning wallet with a single LWS breaks non-freezability, meaning they can refuse to hand over your channel states when recovering.
+
+To prevent this, ensure your users can [manually backup their channel]({{ '//guide/daily-spending-wallet/backup-and-recovery/manual-backup/' | relative_url }}) states or use multiple LWSs for backups.
+
+{% include tip/close.html %}
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/Backups.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/Backups@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Backups-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Backups-mobile@2x.png"
+   alt-text = "How a LWS backups a users channel state as they change"
+   width = 800
+   height = 348
+%}
+
+### Lightning address
+
+Lightning [payment requests]({{ '/guide/payments/request/payment-request-formats/' | relative_url }}) are not very human-readable and are mostly [single-use]({{ '/guide/daily-spending-wallet/requesting/#single-use-payment-requests' | relative_url }}).
+
+A [Lightning address](https://lightningaddress.com/) LWS service gives users a human-readable address that can be re-used to receive lightning payments. An example is bosch@bitcoin.design.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/Lightning-address.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/Lightning-address@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Lightning-address-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Lightning-address-mobile@2x.png"
+   alt-text = "Examples of lightning addresses"
+   width = 800
+   height = 348
+%}
+
+### Receive payments offline
+
+Non-custodial lightning wallets need to be online to receive payments. [Daily spending wallets]({{ '/guide/daily-spending-wallet/' | relative_url }}), in particular, are regularly offline, so receiving payments can be difficult.
+
+LWSs can hold onto a payment (without assuming custody) and forward it to the user once they come online. A product that offers this for users is [Greenlight](https://blockstream.com/lightning/greenlight/).
+
+This service is usually required to be offered alongside an LSP. Read more about this [here](https://medium.com/breez-technology/get-ready-for-a-fresh-breez-multiple-apps-one-node-optimal-ux-519c4daf2536).
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/Offline-payment.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/Offline-payment@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Offline-payment-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Offline-payment-mobile@2x.png"
```

Comment:
I'm having trouble parsing one of the sentences in this image. "Is incoming from the lightning network hits the LWS".

Maybe _"Is incoming from the lightning network and hits the LWS"._

"And" being the key word that makes the sentence easier to read.

Reply:
Done :)